pom.xml

```
@@ -18,16 +18,16 @@
   <licenses>
     <license>
       <name>The MIT license</name>
-      <url>http://www.opensource.org/licenses/mit-license.php</url>
+      <url>https://www.opensource.org/licenses/mit-license.php</url>
       <distribution>repo</distribution>
     </license>
   </licenses>
 
   <properties>
-    <apacheds.version>2.0.0-M23</apacheds.version>
+    <apacheds.version>2.0.0.AM25</apacheds.version>
     <apacheds.jdbm.version>2.0.0-M3</apacheds.jdbm.version>
-    <jenkins.version>1.651.3</jenkins.version>
-    <jenkins-test-harness.version>2.32</jenkins-test-harness.version>
+    <jenkins.version>2.60.3</jenkins.version>
```

You could use a much more recent version of Jenkins than that. How about:

```suggestion
    <jenkins.version>2.138.4</jenkins.version>
```

---

Not sure what the support lifecycle for jenkins. Picked this version just for java 8.
What about how others feel about bumping it again. But I think you have to bump it again for java11